Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -45,16 +45,34 @@ function getCommandOutputFormat(kubecommand: string) : string[] {
 function getCommandConfigurationFile() : string[] {
     var args: string[] =[];
     var useConfigurationFile : boolean  =  tl.getBoolInput("useConfigurationFile", false);
-    if(useConfigurationFile) {
-        var configurationPath = tl.getInput("configuration", true);
-        if(configurationPath && tl.exist(configurationPath))
-        {
-            args[0] = "-f";
-            args[1] = configurationPath;
-        }
-        else
-        {
-           throw new Error(tl.loc('ConfigurationFileNotFound', configurationPath));       
+    if (useConfigurationFile) {
+        var configurationOption:string = tl.getInput("configurationLocation", true);
+        switch(configurationOption){
+            case "configuration" : {
+                var configurationPath = tl.getInput("configuration", true);
+                if (configurationPath && tl.exist(configurationPath)) {
+                    args[0] = "-f";
+                    args[1] = configurationPath;
+                }
+                else {
+                    throw new Error(tl.loc('ConfigurationFileNotFound', configurationPath));
+                }
+                break;
+            }
+            case "inlineConfiguration" : {
+                var inlineConfiguration: string = tl.getInput("inlineConfiguration", false);
```

Comment:
it should be `tl.getInput("inlineConfiguration", true)` should be a required field